doc/api/crypto.md

```diff
@@ -1784,6 +1784,12 @@ otherwise `err` will be `null`. By default, the successfully generated
 `derivedKey` will be passed to the callback as a [`Buffer`][]. An error will be
 thrown if any of the input arguments specify invalid values or types.
 
+The `digest` should either be a string value or null:
```

Comment:
I still believe that mentioning 'string value' here is redundant since it is stated earlier in the function description that `digest` is an HMAC digest algorithm, which implicitly means a string value.